packages/arb-token-bridge-ui/src/hooks/useClaimWithdrawal.ts

```diff
@@ -32,15 +33,18 @@ export function useClaimWithdrawal(): UseClaimWithdrawalResult {
     setIsClaiming(true)
 
     try {
+      if (!signer) {
+        throw 'Signer is undefined'
```

Comment:
we can use Toast for all the error messages later, but for now i don't see an immediate need for this one because Signer would only be undefined when the user deliberately connected a watch-only address to the site